scripts/install

```diff
@@ -85,9 +85,15 @@ pushd "$install_dir"
 
 	# Enable LDAP Auth addon
 	ynh_exec_as "$app" "php$phpversion" bin/console.php addon enable ldapauth
-
 popd
 
+# add the path to the pidfile (we can't do that using the autoinstall)
+# in fact we can using 'autoinstall -f <prepared.config.php>' but i'm tired
```

Comment:
we definively would dig that
the local.config.php file is overwritten by autoinstall so it's useless